xprom/semconv/kafka.go

```diff
@@ -0,0 +1,43 @@
+package semconv
+
+// System names.
+const (
+	SystemKafka = "kafka"
+)
+
+// Metric names.
+// https://github.com/open-telemetry/semantic-conventions/blob/v1.26.0/docs/messaging/messaging-metrics.md
```

Comment:
Is it feasible to re-use [semconv](https://pkg.go.dev/go.opentelemetry.io/otel/semconv@v1.28.0) from otel?

Reply:
`semconv` uses `attribute.Key`. i want to avoid adding otel dependency in this module